lib/scrabble_master.rb

```diff
@@ -1,2 +1,7 @@
 require_relative "scrabble"
 require_relative "player"
+require_relative "tilebag"
+require_relative "dictionary"
+require_relative "game"
+
+play_scrabble
```

Comment:
If you have time (I know we've move on already), I'd recommend double checking that win condition. I did manage to get to 100 points in my rspec run, but it kept asking me to play after that...